roles/ceph-ansible/tasks/main.yml

```diff
@@ -0,0 +1,180 @@
+    regexp: '^#cluster: ceph.*'
+    line: 'cluster: test_name'
+
+- name: Set automated fsid generation
+  lineinfile:
+    dest: "{{ home_dir }}/ceph-ansible/group_vars/all.yml"
+    regexp: '^#generate_fsid: true.*'
+    line: 'generate_fsid: true'
+
+# Disabled because this list in directly in ceph-ansible
+#
+#- name: Set package dependencies for CentOS
+#  lineinfile:
+#    dest: "{{ home_dir }}/ceph-ansible/group_vars/all.yml"
+#    regexp: '^#centos_package_dependencies:.*'
+#    line: 'centos_package_dependencies:'
+#  when: ansible_distribution == 'CentOS'
+#
+#- name: Set package dependencies for CentOS - packages
+#  lineinfile:
+#    dest: "{{ home_dir }}/ceph-ansible/group_vars/all.yml"
+#    insertafter: '^centos_package_dependencies:'
+#    line: "  - {{ item }}"
+#    regexp: "^centos_package_dependencies:\\\n.*  - {{ item }}"
+#  with_items: ["libselinux-python", "python-setuptools", "ntp", "epel-release",
+#              "yum-plugin-priorities.noarch", "hdparm", "python-pycurl"]
+#  when: ansible_distribution == 'CentOS'
+#
+#- name: Set package dependencies for RHEL
+#  lineinfile:
+#    dest: "{{ home_dir }}/ceph-ansible/group_vars/all.yml"
+#    regexp: '^#redhat_package_dependencies:.*'
+#    line: 'redhat_package_dependencies:'
+#  when: ansible_distribution == 'RedHat'
+#
+#- name: Set package dependencies for RHEL - packages
+#  lineinfile:
+#    dest: "{{ home_dir }}/ceph-ansible/group_vars/all.yml"
+#    insertafter: '^redhat_package_dependencies:'
+#    line: "  - {{ item }}"
+#    regexp: "^redhat_package_dependencies:\\\n.*  - {{ item }}"
+#  with_items: ["python-setuptools", "ntp", "hdparm", "python-pycurl"]
+#  when: ansible_distribution == 'RedHat'
+
+
+- name: Set Ceph origin 'distro' which means that repos are already set on nodes
+  lineinfile:
+    dest: "{{ home_dir }}/ceph-ansible/group_vars/all.yml"
+    regexp: "^#ceph_origin: 'upstream'.*"
+    line: "ceph_origin: 'distro'"
+
+- name: Set client authentication
+  lineinfile:
+    dest: "{{ home_dir }}/ceph-ansible/group_vars/all.yml"
+    regexp: "^#cephx: true.*"
+    line: 'cephx: true'
+
+- name: Set monitor interface to ansible_default_ipv4['interface']
+  lineinfile:
+    dest: "{{ home_dir }}/ceph-ansible/group_vars/all.yml"
+    regexp: "^#monitor_interface: interface.*"
+    line: "monitor_interface: {{ ansible_default_ipv4['interface'] }}"
+
+- name: Set journal_size to 5 GB
+  lineinfile:
+    dest: "{{ home_dir }}/ceph-ansible/group_vars/all.yml"
+    regexp: "^#journal_size: 0.*"
+    line: 'journal_size: 5120'
+
+- name: Set public_network to ansible_default_ipv4['address']/16
+  lineinfile:
+    dest: "{{ home_dir }}/ceph-ansible/group_vars/all.yml"
+    regexp: "^#public_network: 0.0.0.0/0.*"
+    line: "public_network: {{ ansible_default_ipv4['address'] }}/16"
+
+# edit mons.yml file
+
+# Disabled on CentOS because calamari is not available in SIG
+- name: Enable calamari on mons
```

Comment:
Ceph-ansible install and run calamari on all mons. Console 2 has no problem with this. It is OK to create user in Calamari only on one instance after installation for import purpose.